src/shared/Core/Trace2Message.cs

```diff
@@ -329,6 +331,8 @@ public class ChildExitMessage : Trace2Message
     public int Code { get; set; }
 
     [JsonProperty("t_rel")]
+    public double RelativeTime { get; set; }
+
     public double ElapsedTime { get; set; }
```

Comment:
This doesn't have a `JsonProperty` anymore? Is `ElapsedTime` still relevant?

Reply:
Yep, we need it to correctly build the span for the Performance format target. However, it's not needed for the Event format, which is why there is no Json attribute.